core/src/main/java/org/mule/util/SplashScreen.java

```diff
@@ -68,7 +80,31 @@ protected void doBody(String line)
     protected void doFooter(MuleContext context)
     {
         // default reserved for mule core info
-    }    
+    }
+
+    protected void listItems(Collection<String> items, String description)
+    {
+        if (!items.isEmpty())
+        {
+            doBody(description);
```

Comment:
This is the current method to add the message from SplashScreen. I've modified the ArtifactSplashScreen ones to createMessage though.